test/workers/schedule_deletion_suspended_accounts_worker_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ScheduleDeletionSuspendedAccountsWorkerTest < ActiveSupport::TestCase
+  setup do
+    @accounts = {to_delete: [], not_to_delete: []}
+
+    # Provider, suspended long ago. To be marked as deleted
+    account = FactoryGirl.create(:simple_provider, state: 'suspended')
```

Comment:
https://github.com/3scale/porta/pull/17/files#r229401670

or maybe even better:
```
@accounts[:to_delete] << FactoryGirl.create(:simple_provider, state: 'suspended', updated_at: Account::States::TIME_TO_SCHEDULE_FOR_DELETION.ago)
```

Reply:
That's what I tried first XD but it doesn't work 😞